src/main/java/com/jmb/springfactory/model/dto/CommentDto.java

```diff
@@ -19,9 +22,11 @@
 
   @NotNull
   private UserDto author;
+  
+  private WorkGroupDto group;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "group" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)